docs/data/date-pickers/fields/fields.md

```diff
@@ -37,25 +37,29 @@ The single input range fields are not supported on the range pickers yet (you ca
 
 The field components have an internal state to update the visible value.
 
-It will only call the `onChange` callback when all the sections of the modified date are filled.
+It will only call the `onChange` callback when:
+
+- all the sections of the modified date are now filled
+- all the sections of the modified date are now empty (will return `null` for the modified date)
+- all the sections of the modified date were filled, and the user cleaned the content of one section
```

Comment:
```suggestion
- all the sections of the modified date were filled, and the user cleaned the content of one section (will return `Invalid date` for modified date)
```
And maybe also add a point of firing the same when starting to enter date in an empty field? 🤔